.gitignore

```diff
@@ -34,3 +34,5 @@ node_modules
 # Optional REPL history
 .node_repl_history
 
+# Nyc test output
+.nyc_output
```

Comment:
@cupidcoder Is this part of setting up unit testing?

Reply:
@paradisekelechi not directly but after I ran tests, it generated the .nyc_output folder which I didn't want to be pushed to the online repo